src/invoices/InvoicesList/Invoice/InvoiceDetailsLayer.js

```diff
@@ -19,17 +21,44 @@ class InvoiceDetailsLayer extends Component {
       fetch: false,
     },
     query: {},
+    user: {
+      ...USERS,
+      accumulate: true,
+      fetch: false,
+    },
   });
 
   static propTypes = {
     match: ReactRouterPropTypes.match,
     mutator: PropTypes.object.isRequired,
     onClose: PropTypes.func.isRequired,
     onEdit: PropTypes.func.isRequired,
+    parentResources: PropTypes.object.isRequired,
     resources: PropTypes.object.isRequired,
     showToast: PropTypes.func.isRequired,
   }
 
+  state = {
+    user: [],
```

Comment:
array?

Reply:
yes, an array of records (in this case with only one record)